src/modules/address-book/components/AddressBookAddEdit.vue

```diff
@@ -212,7 +212,7 @@ export default {
       }
     },
     setAddress(value) {
-      this.addressToAdd = value;
+      this.addressToAdd = value ? value : '';
```

Comment:
is it going to add  '' as the address?

Reply:
only if value is falsy which cascades to the lowercase error since the address isn't a string anymore